webpack/stories/docs/APIMiddleware.md

```diff
@@ -1,48 +1,59 @@
 # API Middleware
 
 Instead of each component handling API calls in the same way we have the API Middleware that will handle it instead.
-Current use as an action:
 
-API_OPERATIONS is defined in 'webpack/assets/javascripts/react_app/redux/API.js'
-or 'foremanReact/redux/API' for plugins
+the API Actions can be imported from 'webpack/assets/javascripts/react_app/redux/API/index.js'
+or 'foremanReact/redux/API/index.js' for plugins
+
+currently the `get` action is available, and soon we will support all of the API operations.
+
+## How to use it in your action
+
 ```js
-import { API_OPERATIONS } from ...;
-const someAction = url => ({
-  type: API_OPERATIONS.GET,
-  key: COMPONENT_NAME,
-  url,
-});
+import { get } from '../../redux/API';
+
+const someAction = url =>
+  get({
+    key: MY_SPECIAL_KEY, // this will be used later to identify your API call, so keep it unique.
+    url: url,
```

Comment:
can be just `url,` to avoid eslint error when copying

Reply:
I documented it this way so it would be more readable for non js develooers too, but I agree with you that eslint will raise an error on it.